### DISCUSSION TITLE
Multiple crashes since the new update

### DISCUSSION THREAD

**Discussion by Masenk0** [Raytwo/ARCropolis#595] [Issues]
I have had multiple crashes since the new ARCropolis update. Mostly during the smash ball loading screens. I've had 2 or 3 crashes from not rematching online to go back to the character select screen, I have had one crash at the loading screen when queueing in with a character on elite smash, and I have had one crash at the loading screen when backing out of the online training room going back to the character select screen.

**Comment by Visualwit** [comment]
Same here. Installing the newest release still did not help me either. My game crashes at the switch logos!

**Comment by AntiAqua** [comment]
I am on the following:

4.0.5 Arcropolis and Atmosphere 1.7.1(Still on Switch firmware 18.1.0). I am also using the latest Skyline as well as the latest Smashline 2 plugin. 

I'm noticing crashes after completing any game and going through the results screen, the following screen stays black and forces the software to close. This has happened about 6 times in a row now, and previously, it was only happening maybe 1 in 5 games. I scrapped all of my mods to try and locate exactly which mod was causing this, but I'm nearly certain it is an issue with Arcropolis due to that weird behavior. It crashed 1 in 5 games about three times, and then it evolved into crashing EVERY time. Is Arcropolis storing cache or something that is causing this? 

If it would help, I can provide any logs or provide a zip with every mod I'm using so you could reproduce this issue(it isn't big as I scrapped my mods to try and figure this out). 

Edit: Added Atmosphere crash report if this helps at all
[01728869128_01006a800016e000.log](https://github.com/user-attachments/files/17357992/01728869128_01006a800016e000.log)

Edit2: Seems to be related with a specific couple stage mods after testing for many many hours. Unsure if it's an Arcropolis problem or not though. The stage in question was perfectly functional during 13.0.2 of Smash. But is now causing issues in 13.0.3. Will keep editing this post with whatever details I come up with in the future should I find any.

**Comment by AntiAqua** [comment]
Edit3: Redacted as it was an issue with a plugin not related to Arcropolis.

**Comment by Haxers007** [comment]
I am currently having the same issue after updating to Arcropolis 4.0.6. I had recently just updated atmosphere to 1.8.0 (Switch firmware 19.0) with Smash version 13.0.3. When launching Smash - it's just stuck at the Nintendo Switch loading screen and will then give out the error message "The software was closed because an error occurred" shortly afterwards. Confirmed and tested all plugins should be working correctly.

Any ideas?

**Reply by Masenk0** [reply]
The one time this happened to me a while ago I simply reinstalled Atmosphere and Arc. If you've already done that and nothing worked then I'm not sure myself. I updated a couple hours after the Atmosphere update came out and everything worked fine for me.
